e2e/tests/functional/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -555,6 +555,36 @@ test.describe('Example Imagery in Time Strip', () => {
     // Navigate to timestrip
     await page.goto(timeStripObject.url);
   });
+
+  test.describe('Example Imagery in Flexible layout', () => {
```

Comment:
Linter errors here. We disallow nested `describe`s for e2e tests. Let's add this test to the existing suite [here](https://github.com/nasa/openmct/pull/7308/files#diff-359082dfca47be1f7378e6ee9d63fd5b7e1c57092365f1a2c8ca0684d559cb95R461).

Let's also add it to the [imagery in display layout suite](https://github.com/nasa/openmct/pull/7308/files#diff-359082dfca47be1f7378e6ee9d63fd5b7e1c57092365f1a2c8ca0684d559cb95R302), since the same behavior should work there as well